Documentation/MRTK_PackageContents.md

```diff
@@ -85,6 +110,66 @@ The optional Microsoft.MixedRealityToolkit.Unity.Examples package includes demon
 | | StandardAssets | Common assets shared by multiple demo scenes. |
 
+## Unity Package Manager
+
+When using Unity 2019.4, or newer, the Microsoft Mixed Reality Toolkit is available using the Unity Package Manager. The following sections describe the contents of these packages.
+
+- [Foundation](#foundation-upm)
+- [Standard Assets](#standard-assets-upm)
+- [Extensions](#extensions-upm)
+- [Tools](#tools-upm)
+- [Test Utilities](#test-utilities-upm)
+- [Examples](#examples-upm)
+
+### Foundation (UPM)
+
+The Unity Package Manager version of the MRTK foundation package (`com.microsoft.mixedreality.toolkit.foundation`) contains most of the same content as the [asset package version](#foundation). 
+
+One notable exception is the MRTK/StandardAssets folder. This folder has been moved into the new [standard assets package](#standard-assets-upm).
+
+> [!Note]
+> When importing the foundation package using the Unity Package Manager, the standard assets package will also be imported.
+
+### Standard Assets (UPM)
+
+The standard assets package (`com.microsoft.mixedreality.toolkit.standardassets`) is exclusive to the MRTK Unity Package Manager distribution. 
+
+| Folder | Component | Description |
+| --- | --- | --- |
+| MRTK/Examples | | |
+| | Audio | Sound files that provide common user interface action acknowledgement. |
+| | Fonts | Font files that closely match the typeface used in the Microsoft HoloLens and Windows Mixed Reality shell experiences. |
+| | FontsSDFTextures | Textures used to produce high quality fonts when scaled. |
+| | Icons | Common icons used in the Mixed Reality Toolkit. |
+| | Materials | Basic materials (ex: solid and translucent colors) that use the MRTK shaders. |
+| | Shaders | Collection of shaders optimized for mixed reality devices. |
+| | Textures | Common textures used in the Mixed Reality Toolkit. |
+
+> [!Note]
+> Standard assets is a stand alone component that can be used independently from the other MRTK packages. It is required by the [foundation](#foundation-upm) and will be included when any MRTK package is imported.
+
+### Extensions (UPM)
+
+The extensions UPM package (`com.microsoft.mixedreality.toolkit.extensions`) contains the same content as the [extensions asset package](#extensions). 
+
+> [!Note]
+> When importing the extensions UPM package, sample scenes are not imported by default.
+
+### Tools (UPM)
+
+The tools UPM package (`com.microsoft.mixedreality.toolkit.tools`) contains the same content as the [tools asset package](#tools). 
+
+### Test Utilities (UPM)
+
+The test utlities UPM package (`com.microsoft.mixedreality.toolkit.testutilities`) contains the same content as the [test utiltiies asset package](#test-utilities). 
+
+### Examples (UPM)
+
+The examples UPM package (`com.microsoft.mixedreality.toolkit.examples`) contains the same content as the [examples asset package](#examples). 
+
+> [!Note]
+> When importing the examples UPM package, sample scenes are not imported by default.
```

Comment:
do you think we should make this clearer by writing adding something like: When importing the examples UPM package, sample scenes are not imported by default and need to be explicitely enabled in the import dialog.

just to avoid confusion that scenes might not be part of the package in UPM?